test/minikube/001-kube.bats

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bats
+#
+# Tests of podman kube commands with minikube
+#
+
+load ../system/helpers.bash
+
+function setup(){
+    # only set up the minikube cluster before the first test
+    if [[ "$BATS_TEST_NUMBER" -eq 1 ]]; then
+        minikube start
+        echo $output >&2
+    fi
+    # wait for a bit so all the services in the cluster come up
+    sleep 10
+}
+
+function wait_for_pods_to_start(){
```

Comment:
Also belongs in your own local helpers.bash

Reply:
fixed